metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/params_linux.py

```diff
@@ -195,7 +195,7 @@
 snort_index_path = tmp_dir + "/snort_index.template"
 yaf_index_path = tmp_dir + "/yaf_index.template"
 error_index_path = tmp_dir + "/error_index.template"
-meta_index_path = tmp_dir + "/meta_index.mapping"
+meta_index_path = tmp_dir + "/meta_index.template"
```

Comment:
I renamed `meta_index.mapping` to `meta_index.template` to be consistent with the other templates.